Traceback (most recent call last):
  File "F:\DSC_B_2024\train.py", line 251, in <module>
    main()
  File "F:\DSC_B_2024\train.py", line 238, in main
    train_data = Sacarsm(mode='train', text_name=args.text_name, limit=None)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\DSC_B_2024\dataset.py", line 13, in __init__
    self.data = self.load_data(mode, limit)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\DSC_B_2024\dataset.py", line 24, in load_data
    for data in dict_datas.values():
                ^^^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'values'
